app/models/matchParticipants.py

```diff
@@ -0,0 +1,12 @@
+from app.models import*
+from app.models.user import User
+from app.models.event import Event
+
+from app.models.outsideParticipant import OutsideParticipant
+
+class EventOutsideParticipants(baseModel):
```

Comment:
This file doesn't match the name of the class

Reply:
resolved